crates/dojo-test-utils/build.rs

```diff
@@ -8,7 +8,12 @@ use scarb::core::Config;
 use scarb::ops;
 use scarb::ui::Verbosity;
 
+#[allow(unreachable_code)]
```

Comment:
lmk if its cool to allow unreachable code here, there was an option to just have an empty main in the case of `skip-build`, but that would just have clippy upset about unused dependencies